base/server/src/com/netscape/cms/authentication/SharedSecret.java

```diff
@@ -182,6 +182,13 @@ public void init(String name, String implName, AuthManagerConfig config)
             mShrTokAttr = DEF_SharedToken_ATTR;
         }
 
+        boolean useOAEP = cs.getBoolean("ca.use_oaep_rsa_key_wrap",false);
+        logger.debug(method + " ca.use_oaep_rsa_key_wrap",false);
```

Comment:
There are references to this class from CS.cfg files in non-CA subsystems, so I don't think we want to use a parameter with `ca.` prefix.